tap_logic4/streams.py

```diff
@@ -648,7 +648,7 @@ class InvoiceRowsStream(Logic4Stream):
 
     def prepare_request_payload(self, context, next_page_token):
         payload = super().prepare_request_payload(context, next_page_token)
-        payload["InvoiceId"] = context["InvoiceId"]
+        payload["OrderId"] = context["InvoiceId"]
```

Comment:
this line should be like below, as the context key has been changed in the child context
`payload["OrderId"] = context["InvoiceId"]`